### PR TITLE
Compatibility testing

### PR DESCRIPTION
**Pull request by C0deJack**
Pull request includes the following:
changes to styling (see commits for details)
addition of new icons
changed icons to greyscale
addition of gifs to overcome video play issues with safari

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -116,48 +122,49 @@ h3 {
...
-    min-height: 390px;
-    background-color: #EAECEE;
+    background-color: #FFDD30;
     position: relative;
 }
 
 section div {
-    padding: 70px 0 0;
-    font-size: 1.5rem;
+    padding: 70px 70px 0;
```
Comment (nsantiagoblair) [inline styles.css:130]: 3 values in padding?

**Review by nsantiagoblair — COMMENTED** [review]
You have lots of absolutely positioned elements which might have been able to be positioned without taking them out of the document flow.

Looks good though  😺

File: index.html [hunk 2]
```diff
@@ -36,44 +38,45 @@ <h3>jackwoodpearce@gmail.com</h3>
...
+                <h3>The Pilot Shop</h3>
                 <p>This was a two day build of a shop home page to enhance our skill to produce a responsive website.</p>
             </div>
         </article>
         <article>
             <div class="img-holder"><img src="assets/mayden-logo.jpg" alt="mayden academy logo"></div>
             <div class="txt-holder">
-                <h4>Mayden Logo HTML & CSS Only Challenge</h4>
-                <p>This was a two day build of a shop home page to enhance our skill to produce a responsive website.</p>
+                <h3>Mayden Logo HTML & CSS Only</h3>
```
Comment (nsantiagoblair) [inline index.html]: I think you might need to escape & with `&amp;`

**Review by nsantiagoblair — COMMENTED** [review]
Found an unescaped &

File: styles.css (hunk 1 above)
Comment (C0deJack) [reply styles.css:130]: yeah this is to say padding top left and right 70px and no padding on the bottom.

File: index.html (hunk 2 above)
Comment (C0deJack) [reply index.html]: Nice spot. i'll change this

**Review by nsantiagoblair — APPROVED** [review]
I find your padding confusing. I'd have gone for 1, 2 or 4 values but I think it's just a matter of preference.

**Pull request merged**